packages/yourloops/components/header-bars/patient-nav-bar-tabs.tsx

```diff
@@ -0,0 +1,153 @@
+import { Theme } from '@mui/material/styles'
+import Button from '@mui/material/Button'
+import GetAppIcon from '@mui/icons-material/GetApp'
+import ChartType from '../../enum/chart-type.enum'
+
+interface PatientNavBarTabsProps {
+  chartType: string
+  prefixURL: string
+  onClickDashboard: MouseEventHandler<HTMLAnchorElement>
+  onClickTrends: MouseEventHandler<HTMLAnchorElement>
+  onClickDaily: MouseEventHandler<HTMLAnchorElement>
+  onClickPrint: MouseEventHandler<HTMLButtonElement>
+}
+
+const TAB_HEIGHT = '48px'
+
+const styles = makeStyles()((theme: Theme) => {
+  return {
+    root: {
+      minHeight: TAB_HEIGHT,
+      height: TAB_HEIGHT
+    },
+    tabs: {
+      boxShadow: theme.shadows[3],
+      backgroundColor: theme.palette.common.white
+    },
+    tab: {
+      fontWeight: 'bold',
+      marginRight: theme.spacing(5),
+      textTransform: 'capitalize',
+      fontSize: theme.typography.htmlFontSize,
+      color: 'var(--text-base-color)'
+    }
+  }
+})
+
+export const PatientNavBarTabs: FunctionComponent<PatientNavBarTabsProps> = (
+  {
+    prefixURL = '',
+    ...props
+  }) => {
+  const {
+    chartType,
+    onClickDashboard,
+    onClickTrends,
+    onClickDaily,
+    onClickPrint
+  } = props
+
+  const { t } = useTranslation('yourloops')
+
+  const { classes } = styles()
+
+  const selectedTab = (): number => {
+    switch (chartType) {
+      case ChartType.Dashboard:
+        return 0
+      case ChartType.Daily:
+        return 1
+      case ChartType.Trends:
+        return 2
+    }
+  }
+
+  return (
+    <Box className={classes.tabs} width="100%" paddingLeft={7}>
+      <Box display="flex">
+        <Tabs value={selectedTab()} classes={{
+          root: classes.root
+        }}>
+          <Tab
+            className={classes.tab}
+            data-testid="dashboard-tab"
+            href={`${prefixURL}/dashboard`}
+            iconPosition="start"
+            label={t('dashboard')}
+            icon={<DashboardOutlinedIcon />}
+            onClick={onClickDashboard}
+            classes={{
+              root: classes.root
+            }}
+          />
+          <Tab
+            className={classes.tab}
+            data-testid="daily-tab"
+            href={`${prefixURL}/daily`}
+            iconPosition="start"
+            label={t('daily')}
+            icon={<Today />}
+            onClick={onClickDaily}
+            classes={{
+              root: classes.root
+            }}
+          />
+          <Tab
+            className={classes.tab}
+            data-testid="trends-tab"
+            href={`${prefixURL}/trends`}
+            iconPosition="start"
+            label={t('trends')}
+            icon={<TrendingUp />}
+            onClick={onClickTrends}
+            classes={{
+              root: classes.root
+            }}
+          />
+        </Tabs>
+        <Box marginLeft="auto" marginRight={3} display="flex" justifyContent="center" alignItems="center">
```

Comment:
I suggest a css improvement: add padding on the container instead of margin on the element. I think it's easier to maintain.